source/educators/how-tos/connect_teams_content_groups.rst

```diff
@@ -21,4 +21,19 @@ Connecting Teams to Content Groups
    the list of Teams from the selected group will be shown. Authors may then
    select one or multiple Teams.
 
-     .. image:: /_images/educator_how_tos/teams_within_groups.png
\ No newline at end of file
+     .. image:: /_images/educator_how_tos/teams_within_groups.png
+
```

Comment:
@jesgreco you need to put in the "see also" here! Please don't resolve comments until I've verified you understood my comment.